### PR TITLE
Add paddle mobilenets

### PR DESCRIPTION
**Pull request by eaidova**
TO DO:

- [x] update accuracy refs for mobilenet-v3-small
- [x] update gflops

### CODE REVIEW

**Comment by eaidova** [comment]
Jenkins please retry a build

**Comment by eaidova** [comment]
Jenkins please retry a build

**Comment by eaidova** [comment]
Jenkins please retry a build

File: models/public/mobilenet-v3-large-1.0-224-paddle/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# mobilenet-v3-small-1.0-224-tf
```
Comment (Anna-Mironova) [inline models/public/mobilenet-v3-large-1.0-224-paddle/README.md:1]:
```suggestion
# mobilenet-v3-large-1.0-224-paddle
```

File: models/public/mobilenet-v3-large-1.0-224-paddle/README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+## Use Case and High-Level Description
+
+`mobilenet-v3-small-1.0-224-paddle` is one of MobileNets V3 - next generation of MobileNets,
+based on a combination of complementary search techniques as well as a novel architecture design.
+`mobilenet-v3-small-1.0-224-paddle` is pretrained in Paddle\* framework and targeted for high resource use cases.
```
Comment (Anna-Mironova) [inline models/public/mobilenet-v3-large-1.0-224-paddle/README.md:7]:
```suggestion
`mobilenet-v3-large-1.0-224-paddle` is one of MobileNets V3 - next generation of MobileNets,
based on a combination of complementary search techniques as well as a novel architecture design.
`mobilenet-v3-large-1.0-224-paddle` is pretrained in Paddle\* framework and targeted for high resource use cases.
```

File: models/public/mobilenet-v3-large-1.0-224-paddle/README.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+Expected color order: `BGR`.
+
+## Output
+
+### Original Model
+
+Object classifier according to ImageNet classes, name - `softmax_1.tmp_0`,  shape - `1, 1000`, output data format is `B, C` where:
+
+- `B` - batch size
+- `C` - Predicted probabilities for each class in  [0, 1] range
```
Comment (Anna-Mironova) [inline models/public/mobilenet-v3-large-1.0-224-paddle/README.md:57]:
```suggestion
- `C` - predicted probabilities for each class in [0, 1] range
```

File: models/public/mobilenet-v3-large-1.0-224-paddle/model.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  "mobilenet-v3-small-1.0-224-paddle" is one of MobileNets V3 - next generation of MobileNets,
+  based on a combination of complementary search techniques as well as a novel architecture
+  design. "mobilenet-v3-small-1.0-224-paddle" is pretraind in Paddle framework and targeted for low resource use cases.
```
Comment (Anna-Mironova) [inline models/public/mobilenet-v3-large-1.0-224-paddle/model.yml:18]:
```suggestion
  "mobilenet-v3-large-1.0-224-paddle" is one of MobileNets V3 - next generation of MobileNets,
  based on a combination of complementary search techniques as well as a novel architecture
  design. "mobilenet-v3-large-1.0-224-paddle" is pretraind in Paddle framework and targeted for low resource use cases.
```

File: models/public/mobilenet-v3-small-1.0-224-paddle/README.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
+# mobilenet-v3-small-1.0-224-tf
```
Comment (Anna-Mironova) [inline models/public/mobilenet-v3-small-1.0-224-paddle/README.md:1]:
```suggestion
# mobilenet-v3-small-1.0-224-paddle
```

File: models/public/mobilenet-v3-small-1.0-224-paddle/README.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+Expected color order: `BGR`.
+
+## Output
+
+### Original Model
+
+Object classifier according to ImageNet classes, name - `softmax_1.tmp_0`,  shape - `1, 1000`, output data format is `B, C` where:
+
+- `B` - batch size
+- `C` - Predicted probabilities for each class in  [0, 1] range
```
Comment (Anna-Mironova) [inline models/public/mobilenet-v3-small-1.0-224-paddle/README.md:57]:
```suggestion
- `C` - predicted probabilities for each class in [0, 1] range
```

File: models/public/mobilenet-v3-small-1.0-224-paddle/README.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+### Original Model
+
+Image, name: `x`, shape: `1, 3, 224, 224`, format: `B, C, H, W`, where:
+
+- `B` - batch size
+- `C` - number of channels
+- `H` - image height
+- `W` - image width
+
+Expected color order: `RGB`.
```
Comment (Anna-Mironova) [inline models/public/mobilenet-v3-small-1.0-224-paddle/README.md:37]:
```suggestion
Expected color order: `RGB`.
Mean values - [123.675,116.28,103.53], scale values - [58.395, 57.12, 57.375].
```

File: models/public/mobilenet-v3-large-1.0-224-paddle/README.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+### Original Model
+
+Image, name: `x`, shape: `1, 3, 224, 224`, format: `B, C, H, W`, where:
+
+- `B` - batch size
+- `C` - number of channels
+- `H` - image height
+- `W` - image width
+
+Expected color order: `RGB`.
```
Comment (Anna-Mironova) [inline models/public/mobilenet-v3-large-1.0-224-paddle/README.md:37]:
```suggestion
Expected color order: `RGB`.
Mean values - [123.675,116.28,103.53], scale values - [58.395, 57.12, 57.375].
```

File: models/public/mobilenet-v3-large-1.0-224-paddle/README.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Classification                            |
+| GFlops                          | 0.4565                                    |
+| MParams                         | 5.468                                     |
+| Source framework                | Paddle\*                                  |
+
+## Accuracy
+
+| Metric | Result         |
```
[the remark this answers is not in the document]
Comment (eaidova) [reply models/public/mobilenet-v3-large-1.0-224-paddle/README.md:21]:
```suggestion
| Metric | Result         |
```

**Pull request merged**